src/components/Books.js

```diff
@@ -0,0 +1,13 @@
+import React from 'react';
+
+const Books = () => (
+  <div>
+    <h3>Book list</h3>
+    <button type="button">Remove book</button>
+    <div>
+      <input type="text" placeholder="Add a book" />
+    </div>
+  </div>
+);
+
+export default Books;
```

Comment:
- [ ] We need to set up reusable components:

![image](https://user-images.githubusercontent.com/57421823/151525855-aed7b029-0824-49a2-9cb5-966e19f11cf1.png)

In order to comply with this requirement, the Books component should call other components inside it and render them.

The components that should be called are:

1.  The `Book` component (singular) renders each book:

![image](https://user-images.githubusercontent.com/57421823/151526531-223dd1bb-92ea-439e-8240-4dff4bd528bd.png)

2.  The `CreateNewBook` component, which is essentially a form:

![image](https://user-images.githubusercontent.com/57421823/151526958-5c5ea42f-d8b0-432b-be64-d745c60e73b4.png)

Kindly please create these components as per the instructions above and include them inside the `Books` component.

Here is an example of how can those be included inside your `Books` component:

![image](https://user-images.githubusercontent.com/57421823/151529383-dd2182e0-d45a-4932-a7a1-482a8dc93d25.png)

Note: The form inside your `CreateNewBook` component should have a field for the name of the book, and another for the author!

That way, your components will be reusable, and ready for the data! 💪